index.html

```diff
@@ -23,6 +23,7 @@
       <li><a href="src/web/basics/typography/tests/tests-typography.html">tests-typography.html</a></li>
       <li><a href="src/web/components/global-nav/tests/tests-global-nav-bim.html">tests-global-nav-bim.html</a></li>
       <li><a href="src/web/components/modal/tests/test-modal.html">modal.html</a></li>
+      <li><a href="src/web/components/table/tests/tests-table.html">tests-table.html</a></li>
     </ol>
```

Comment:
@nfiniteset to address running the gh compile command.  I had in fact run this and added the tests-table to the appropriate file :) I think you were referencing in your comments